appveyor/crowdinSync.py

```diff
@@ -0,0 +1,55 @@
+# based on file from https://github.com/jcsteh/osara
+
+import time
+import os
+import glob
+import sys
+import zipfile
+import shutil
+import subprocess
+import requests
```

Comment:
PEP 8 (imports) suggests sorting imports and separating/grouping standard library imports, related third-party imports, local application/library specific imports.

```suggestion
import glob
import os
import shutil
import subprocess
import sys
import time
import zipfile

import requests
```